samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,125 @@
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT` is set to a small value 2.5 ms because the sample doesn't send any data over ACL connection (except Control procedures).
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_BIG_RESERVED_TIME_US` is set to half of the ISO Interval to ensure BIG parameters are selected in a way leaving enough time for ACL and CIG.
+   * :kconfig:option:`CONFIG_BT_CTLR_SDC_CIG_RESERVED_TIME_US` is set to half of the ISO Interval to ensure CIG parameters are selected in a way leaving enough time for ACL and BIG.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/iso_combined_bis_and_cis`
+
+.. include:: /includes/build_and_run_ns.txt
+
+
+Testing
+=======
+
+In addition to the device running this sample, two additional devices are required:
+* CIS Peripheral device that sends data over CIS.
+* BIS Sink device that receives data over BIS.
+Other sample applications can be used as CIS Peripheral and BIS Sink (for example, :ref:`bluetooth_isochronous_time_synchronization`)
+CIS Peripheral device should have the same name as the name set in :kconfig:option:`CONFIG_BT_DEVICE_NAME`.
+This name is used to connect to the correct device.
+
+After programming the sample to the development kit and programming other samples to CIS Peripheral device and BIS sink device, perform the following steps to test it:
+
+1. |connect_terminal_specific|
+#. Reset the kits.
+#. Wait until the sample connected to CIS Peripheral device and established CIS connection.
+#. Observe packets transmitted by CIS Peripheral to be retransmitted over BIS and received by BIS Sink.
+
+
+Sample output
+==============
+
+[00:00:00.454,589] <inf> bt_hci_core: HW Platform: Nordic Semiconductor (0x0002)
+[00:00:00.454,620] <inf> bt_hci_core: HW Variant: nRF53x (0x0003)
+[00:00:00.454,650] <inf> bt_hci_core: Firmware: Standard Bluetooth controller (0x00) Version ... Build ...
+[00:00:00.456,329] <inf> bt_hci_core: Identity: FD:DD:D6:88:AF:D4 (random)
+[00:00:00.456,359] <inf> bt_hci_core: HCI: version 5.4 (0x0d) revision ..., manufacturer 0x0059
+[00:00:00.456,390] <inf> bt_hci_core: LMP: version 5.4 (0x0d) subver ...
+[00:00:00.456,390] <inf> app_main:
+Bluetooth ISO Combined BIS and CIS sample
+
+The sample demonstrates data transfer over Bluetooth ISO
+CIS and BIS using the following topology:
+
+┌------┐     ┌-------┐      ┌--------┐
+|      |ISO  |  CIS  | ISO  |        |
+| CIS  ├----►|Central├-----►|BIS Sink|
+|Periph|CIS  |+ BIS  | BIS  |        |
+|      |     |Source |      |        |
+└------┘     └-------┘      └--------┘
+  (1)           (2)            (3)
+The sample only operates as a device 2: Combined CIS Central + BIS Source
+Please, use other samples as a CIS Peripheral (TX) and BIS Sink (RX) devices.
```

Comment:
Can we list some or any concrete examples that may be used?

Reply:
It is already described in a Readme, I think it is sufficient